<ui version="4.0" >
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Login</string>
  </property>
  <property name="windowIcon" >
   <iconset>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="uiKoo" >
     <property name="cursor" >
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="common.qrc" >:/images/koo.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="uiServer" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QComboBox" name="uiDatabase" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Database:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>uiDatabase</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="pushChange" >
       <property name="text" >
        <string>C&amp;hange</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >
         <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Server:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>pushChange</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="spacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="uiNoConnection" >
       <property name="text" >
        <string>&lt;b>Could not connect to server !&lt;/b></string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="uiUserName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>uiPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="uiPassword" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;User:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>uiUserName</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushAccept" >
       <property name="text" >
        <string>&amp;Accept</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="common.qrc" >
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiPassword</tabstop>
  <tabstop>pushAccept</tabstop>
  <tabstop>pushCancel</tabstop>
  <tabstop>pushChange</tabstop>
  <tabstop>uiDatabase</tabstop>
  <tabstop>uiUserName</tabstop>
 </tabstops>
 <resources>
  <include location="common.qrc" />
 </resources>
 <connections/>
</ui>
